shared/settings/landing/container.js

```diff
@@ -12,11 +12,17 @@ const mapStateToProps = state => {
   const {emails} = state.settings.email
   const {rememberPassphrase} = state.settings.passphrase
   let accountProps
-  if (emails.length > 0) {
+  if (emails) {
```

Comment:
`if (emails?.length)` and remove the `if` on L17

Reply:
I don't think that's good though, because we still want to set 
```
      hasRandomPW: state.settings.passphrase.randomPW,
      onChangeEmail: () => logger.debug('todo'),
      onChangePassphrase: () => logger.debug('todo'),
      rememberPassphrase,
```
even if user has zero emails.

Reply:
Then should it be outside this `if`?